UnityProject/Assets/Scripts/Camera/MoveCamera.cs

```diff
@@ -0,0 +1,50 @@
+using UnityEngine;
+using System.Collections;
+
+public class MoveCamera : MonoBehaviour 
+{
+	public float panSpeed = 1.0f;		
```

Comment:
Indentation out of place everywhere in this file.

Reply:
Riiiight I'm on it, sorry ^^"